src/nvim/getchar.c

```diff
@@ -3281,6 +3041,8 @@ static void mapblock_free(mapblock_T **mpp)
   XFREE_CLEAR(mp->m_desc);
   *mpp = mp->m_next;
   xfree(mp);
+  if(mpp2)
+    *mpp2 = 0;
```

Comment:
Shouldn't this be `NULL`, not `0`?

Anyway (imho) this function should just free the block, if there's something to make NULL somewhere else, it should be handled there.

I say this because in the model I'm working on I need to free maps, and make NULL the place where they're accessed (am I understanding right this is the point of it?), but I'd rather be explicit and do it where I need, rather than passing an address to a function that does something else, so that it can make it NULL as side effect. I hope I made myself clear.